### ISSUE TITLE
[BITV]: Settings navigation was not implemented right

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42599]
![Screenshot from 2024-01-05 17-26-57](https://github.com/nextcloud/server/assets/6078378/d3d16149-b34f-4b25-88a4-6b3d167dfe9c)

- List item does not have a `<ul>, <ol>` parent element without a role, or a role="list"
- Required ARIA parent role not present: tablist

Right implementation should look like:

![Screenshot from 2024-01-05 16-21-34](https://github.com/nextcloud/server/assets/6078378/193d168a-71d4-4629-99d1-ae531a5b9c4a)

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tablist_role

**Comment by ShGKme** [comment]
I don't think this list should be a `tablist` at all. There is no tab panels.

**Comment by JuliaKirschenheuter** [comment]
> I don't think this list should be a tablist at all. There is no tab panels.

Researched and agree

**Comment by szaimen** [comment]
I think files and photos is indeed missing a dependency update:
![image](https://github.com/nextcloud/server/assets/42591237/b59d997c-7a71-4dfc-8c1b-a1f8a54f4a6d)
